.github/workflows/main.yaml

```diff
@@ -0,0 +1,20 @@
+name: Main Ci/CD
+on: [push]
```

Comment:
Push in all branches or some knowing branch? You can set this workflow in different events. 
on:
  push:
    branches: [ "main", "dev", "fix", etc. ]

Reply:
@MariaMaximova, yes, this workflow is intended to run on all branches on push to run basic tests to see if the project is ok. Other actions might have different configuration in the future, for example, build or some sort of deploy